pkg/render/kubecontrollers/kube-controllers.go

```diff
@@ -342,19 +342,19 @@ func kubeControllersRoleEnterpriseCommonRules(cfg *KubeControllersConfiguration)
 			Verbs:     []string{"get", "watch"},
 		},
 		{
-			APIGroups: []string{"projectcalico.org"},
+			APIGroups: []string{"projectcalico.org", "crd.projectcalico.org"},
 			Resources: []string{"deeppacketinspections"},
 			Verbs:     []string{"get", "watch", "list"},
 		},
 		{
 			APIGroups: []string{"crd.projectcalico.org"},
-			Resources: []string{"deeppacketinspections"},
-			Verbs:     []string{"get"},
+			Resources: []string{"deeppacketinspections/status"},
+			Verbs:     []string{"update"},
```

Comment:
We don't need get?

Reply:
This diff view makes it look very confusing.

I added "crd.projectcalico.org" in line 345. So in actuality, the "crd.projectcalico.org" now has added watch and list.

And for packetcaptures, it has added get, list and watch:

![image](https://user-images.githubusercontent.com/15924949/159054921-7d82df79-d8c9-4fe4-ae07-8a452dcbdf10.png)

Reply:
Ohhh right, gotcha. Get for the status is just get on the resource I'm not realising.